blockchain/blockchain-wallet-v4-frontend
# Feat/bank linking

## Description (optional)
This PR adds a new payment method for US users. It's an ACH bank transfer payment method so users can connect their US based bank account to their custodial wallet through the simple buy feature. Very similar to the credit card flow, users will now see their linked bank accounts along side their saved credit cards and bank wire beneficiaries. 

## Testing Steps (optional)
To test this, use the simple buy flow for any crypto and follow the steps to "Add payment method" -> "Link A Bank" once your bank is linked you should be able to select the linked bank account and purchase the crypto very similarly to how you would with a credit card. Also similar to credit cards, there is also functionality to list/delete saved bank accounts in the user's general settings.
